src/misc/msccl/msccl_lifecycle.cc

```diff
@@ -448,18 +452,60 @@ ncclResult_t mscclEnqueueCheck(
     count, dataType, root, peer, op, func, comm, stream,
     &threadLocalStatus.savedSchedulerParams.back()));
 
+  //printf("In msccl enqueue check\n");
+  NCCLCHECK(mscclGetCaptureStatus(stream));
+  size_t nBytes = count * ncclTypeSize(dataType);
+
   switch (threadLocalStatus.groupStatus) {
     case mscclNoGroup:
+#ifdef ENABLE_MSCCLPP
+      if (comm->mscclppCompatible) {
+        /* check if one rank per GPU and graph mode is enabled */
+        if ((nBytes > 0 && nBytes <= comm->mscclpp_threshold) && (threadLocalStatus.captureStatus != mscclNoCapture) && comm->mscclCompatible) {
```

Comment:
Since MSFT did not fix the small message issue yet, we need to have a lower limit of 32B.

Reply:
```suggestion
        if ((nBytes > 32 && nBytes <= comm->mscclpp_threshold) && (threadLocalStatus.captureStatus != mscclNoCapture) && comm->mscclCompatible) {
```

Reply:
Should be `nBytes >= 32`.